### ISSUE TITLE
Installation problem

### ISSUE THREAD

**Issue by bithosthun** [projectsend/projectsend#507]
Dear Developers!

Sorry, I'm not very good speak English.

I would have a problem with the system. We were unable to open the system on the web site because of a 500-page browser failure. We mainly use the master, but the 754 is also installed. Unfortunately, you do not want to load any of them.

Could you help me?

PHP version: 5.6

The problem was first not found on this site, but on the storage space of another service, so we can say it's just a trial.

Thank you very much for your help!

Richard

**Comment by richardradiohun** [comment]
PS: Installed by r754 Softaculous, no error has been posted.

**Comment by lenamtl** [comment]
Hi, 

I have edit your post, for security reason don't post live link .

500 Internal Server Error is a generic error message, given when no more specific message is suitable. 
There are a number of causes for a 500 Internal Server Error to display in a web browser. 

I'm not sure but if you have installed Projectsend over another Projectsend installation this can be the cause, so I suggest you clean the folder and datatbase and start from scratch.

If you still get error after that:
Check if Projectsend is installed (check if the are projectsend file in your folder).
Check if database was created (phpmyadmin from cpanel)
Check if you have error in your PHP log file available from cpanel
You need to make sure that PDO is enabled.
Check folder permissions.
You will need to make some PHP ini settings for max file size ect.
You may need to contact your hosting provider to open a ticket and get support.

**Comment by richardradiohun** [comment]
Dear lenamtl!

Thanks for your response! I've done the steps you are suggesting, but unfortunately still do not want to work. I created a new database, but unfortunately it will appear after installation. I've already tried all the options that came to my mind, but so far, unfortunately, no solution was available. Unfortunately, the 500 error is no longer present, so I can not see what caused the error.
https://prnt.sc/i4gfet

I look forward to your reply.
